tutorials/governance/opengov/create-a-referenda.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Create a Referenda
+description: Learn how to create and submit referenda on the Polkadot network, including using the Whitelisted Caller track for time-sensitive or technical proposals.
+---
+
+# Create a Referenda
+
+## Introduction
+
+This guide outlines the process of proposing public referenda using the Referenda module, also known as OpenGov. Before submitting a referendum, it's crucial to identify the appropriate track and origin for your proposal. To ensure you select the most suitable ones for your referendum, refer to [Origins and Tracks](https://wiki.polkadot.network/docs/learn-polkadot-opengov#origins-and-tracks){target=\_blank}.
+
+## Submit a Preimage
+
+The process of creating a proposal is separate from submitting its [preimage](https://wiki.polkadot.network/docs/glossary#preimage){target=\_blank}. This separation is crucial due to large preimages' potentially high storage costs. By allowing preimage submission as a separate transaction, it becomes possible for another account to submit the preimage on your behalf and cover the associated fees.
+
+The example below demonstrates the creation of a preimage to propose and approve the spending of treasury funds.
+
+To submit a preimage, follow these steps:
+
+1. Visit [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpolkadot.api.onfinality.io%2Fpublic-ws#/explorer), navigate to the **Governance** dropdown and select the **Preimages** option
+   
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-1.webp)
+
+2. Click in the **+ Add preimage** button
+   
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-2.webp)
+
+3. Construct the extrinsic you want to execute and submit the preimage:
+    1. Select the **treasury** pallet
+    2. Choose the **spendLocal** extrinsic
+    3. Fill in the parameters:
```

Comment:
Maybe this is nitpicking, but I think that moving step 'C' to just in the middle of "amount" and "beneficiary" would be clearer